pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -211,6 +214,51 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 		}
 	}
 
+	scanSetting := &compliancev1alpha1.ScanSetting{}
+	if instance.SettingsRef != nil {
+		ssb := instance.DeepCopy()
+		err = r.Client.Get(context.TODO(), types.NamespacedName{Name: instance.SettingsRef.Name, Namespace: request.Namespace}, scanSetting)
+		if err != nil {
+			return reconcile.Result{}, fmt.Errorf("Failed to get ScanSetting: %s", err)
+		}
+		if scanSetting.ComplianceSuiteSettings.Suspend {
+			log.Info("Suspending ScanSettingBinding", "ScanSettingBinding", ssb.Name)
+			ssb.Status.SetConditionSuspended()
+			ssb.Status.Phase = compliancev1alpha1.ScanSettingBindingPhaseSuspended
+			err := r.Client.Status().Update(context.TODO(), ssb)
+			if err != nil {
+				return reconcile.Result{}, fmt.Errorf("Failed to update ScanSettingBinding status: %s", err)
+			}
+			cs := compliancev1alpha1.ComplianceSuite{}
+			err = r.Client.Get(context.TODO(), types.NamespacedName{Name: instance.Name, Namespace: request.Namespace}, &cs)
+			// If the ComplianceSuite doesn't exist, then we should
+			// return and not requeue the request because the
+			// ScanSettingBinding is being created from a suspended
+			// state and we don't want to create the
+			// ComplianceSuite or kick off any scans, yet. If or
+			// when the ScanSetting is activated, we will proceed
+			// like normal and create the ComplianceSuite and its
+			// ComplianceScans.
+			if errors.IsNotFound(err) {
+				return reconcile.Result{}, nil
+			}
+
+		} else if !scanSetting.ComplianceSuiteSettings.Suspend {
+			ssb.Status.SetConditionReady()
```

Comment:
Analogous to the log that the `ScanSettingBinding` was suspended, I miss a log entry that the SSB  was resumed.
`log.Info("Resuming ScanSettingBinding", "ScanSettingBinding", ssb.Name)`

Reply:
Good find.